ncdomain/convert.go

```diff
@@ -353,10 +355,23 @@ func ParseValue(name, jsonValue string, resolve ResolveFunc, errFunc ErrorFunc)
 	var rv interface{}
 	v := &Value{}
 
-	err := json.Unmarshal([]byte(jsonValue), &rv)
-	if err != nil {
-		errFunc.add(err)
-		return
+	errJSON := json.Unmarshal([]byte(jsonValue), &rv)
+	if errJSON != nil {
+		valueBytes, errHex := hex.DecodeString(jsonValue)
+		if errHex != nil {
+			errFunc.add(errJSON)
+			errFunc.add(fmt.Errorf("failed to decode hex (after JSON decoding failed): %w", errHex))
+			return
+		}
```

Comment:
What's the intent of decoding as hex? Namecoin values are never hex-encoded in the blockchain by convention. Is there some btcd-related API that's returning a hex string here?

Reply:
The CBOR data is encoded in hex format.

Reply:
Not sure I follow -- where is it hex-encoded? It's not hex-encoded in the blockchain (that would double the storage costs for no reason).

Reply:
I thought when submitting `cbor data` you convert it to `hex` when pass it as a value from a name